pkg/api/entries.go

```diff
@@ -438,3 +421,36 @@ func SearchLogQueryHandler(params entries.SearchLogQueryParams) middleware.Respo
 
 	return entries.NewSearchLogQueryOK().WithPayload(resultPayload)
 }
+
+// Attempt to retrieve a UUID from a backend tree
+func RetrieveUUID(params entries.GetLogEntryByUUIDParams, uuid string, tid int64) (middleware.Responder, error) {
+	ctx := params.HTTPRequest.Context()
+	hashValue, _ := hex.DecodeString(uuid)
```

Comment:
We should check this error in case someone passes in a malformed UUID?

Reply:
I just moved this line of code from where it was earlier, but checking it seems like a good thing to do.

Reply:
thanks ! :)